src/Avatar.js

```diff
@@ -1,31 +1,14 @@
-import React, {useState} from 'react';
+import React from 'react';
 
 import './App.css';
 
-export function Avatar({avatar, name, onClickAvatar}) {
-    const [isErrorLoadImg, setIsErrorLoadImg] = useState(false);
-
-    let initials = name && name.trim().split(' ', 2).map(n => n[0]).join('').toUpperCase();
-
+export function Avatar({avatar, onErrorLoad}) {
     return (
-        <div onClick={onClickAvatar} class="avatar flex-center flex-column">
-            {(avatar && !isErrorLoadImg 
-                && (
-                    <img 
-                        class="avatar-image" 
-                        src={avatar} 
-                        alt=""
-                        onError={() => {setIsErrorLoadImg(true);}}
-                    />
-                )) || (
-                    initials && <div class="avatar-initials flex-center">{initials}</div>
-                ) || (
-                    <>
-                        <div class="symbol-head" />
-                        <div class="symbol-shoulders" />
-                    </>
-                )
-            }
-        </div>
+        <img 
+            class="avatar-image" 
+            src={avatar} 
+            alt=""
+            onError={onErrorLoad}
+        />
```

Comment:
whats the purpose for this component now?
I didn't get me, I want you to move content conditional rendering in the separate component but you moved everything and now Avatar component is super dumb and it does nothing